modules/transcription-service-google-speech-impl/src/main/java/org/opencastproject/transcription/googlespeech/GoogleSpeechTranscriptionService.java

```diff
@@ -604,6 +604,10 @@ boolean getAndSaveJobResults(String jobId) throws TranscriptionServiceException,
     } catch (TranscriptionServiceException e) {
       throw e;
     } catch (Exception e) {
+      if (hasTranscriptionRequestExpired(jobId)) {
+        // Cancel the job and inform admin
+        cancelTranscription(jobId, "Transcription job canceled due to errors");
+      }
```

Comment:
It would help to log an INFO that the job is being cancelled, in addition to the email. Since there are 5 hours of retry logs, this would be the one log brining the closure for the transcription job.

Reply:
Agree